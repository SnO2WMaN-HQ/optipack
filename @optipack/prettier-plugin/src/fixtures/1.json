{
  "name": "prettier-plugin-optipack",
  "version": "0.0.1",
  "author": "SnO2WMaN <sno2wman.official@gmail.com>",
  "license": "MIT",
  "main": "lib/index.js",
  "types": "lib/index.d.ts",
  "keywords": [
    "format",
    "plugin",
    "prettier",
    "prettier-plugin",
    "optipack",
    "package.json"
  ],
  "homepage": "https://github.com/SnO2WMaN-HQ/optipack/tree/master/@optipack/prettier-plugin",
  "bugs": {
    "url": "https://github.com/SnO2WMaN-HQ/optipack/issues"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/SnO2WMaN-HQ/optipack.git",
    "directory": "@optipack/prettier-plugin"
  },
  "scripts": {
    "build": "tsc -b",
    "clean": "tsc -b --clean",
    "test": "jest",
    "test:watch": "jest --watch"
  },
  "peerDependencies": {
    "prettier": "^2.1.1"
  },
  "devDependencies": {
    "typescript": "3.9.7"
  },
  "dependencies": {
    "@optipack/core": "^0.0.1",
    "cosmiconfig": "^7.0.0"
  }
}
