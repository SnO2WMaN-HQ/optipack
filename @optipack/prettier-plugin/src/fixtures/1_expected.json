{
  "name": "prettier-plugin-optipack",
  "version": "0.0.1",
  "license": "MIT",
  "author": "SnO2WMaN <sno2wman.official@gmail.com>",
  "homepage": "https://github.com/SnO2WMaN-HQ/optipack/tree/master/@optipack/prettier-plugin",
  "repository": {
    "type": "git",
    "url": "https://github.com/SnO2WMaN-HQ/optipack.git",
    "directory": "@optipack/prettier-plugin"
  },
  "bugs": {
    "url": "https://github.com/SnO2WMaN-HQ/optipack/issues"
  },
  "keywords": [
    "format",
    "optipack",
    "package.json",
    "plugin",
    "prettier",
    "prettier-plugin"
  ],
  "main": "lib/index.js",
  "types": "lib/index.d.ts",
  "scripts": {
    "build": "tsc -b",
    "clean": "tsc -b --clean",
    "test": "jest",
    "test:watch": "jest --watch"
  },
  "dependencies": {
    "@optipack/core": "^0.0.1",
    "cosmiconfig": "^7.0.0"
  },
  "devDependencies": {
    "typescript": "3.9.7"
  },
  "peerDependencies": {
    "prettier": "^2.1.1"
  }
}
